samples/bluetooth/channel_map_updates/src/main.c

```diff
@@ -0,0 +1,467 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/console/console.h>
+#include <string.h>
+#include <stdlib.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/types.h>
+
+#if defined(CONFIG_USB_DEVICE_STACK)
```

Comment:
why is **new** code being added using the **old, deprecated** USB stack? @jfischer-no @carlescufi